test/functional/gridfs_stream.test.js

```diff
@@ -1232,7 +1226,9 @@ describe('GridFS Stream', function () {
   });
 
   function testResultDoc(specDoc, resDoc, result) {
-    const specKeys = Object.keys(specDoc).sort();
+    const specKeys = Object.keys(specDoc)
+      .filter(key => key !== 'md5')
+      .sort();
```

Comment:
There's new spec tests that omit the md5 property for us, we should use those, but we are also behind on our GridFS sync so I defer that to later work.